src.ts/providers/json-rpc-provider.ts

```diff
@@ -60,6 +60,10 @@ export class JsonRpcProvider extends Provider {
         return this.sendJsonRpc('chunk', [chunkId]);
     }
 
+    async gasPrice(blockId: BlockId|null): Promise<GasPrice> {
```

Comment:
given that `null` is allowed for `blockId` – tests should also check if method works with `null`

Reply:
I know that but do not know the exact price for latest block. We should not suppose it is all the same for '5000', right? so I really want to know how to write that test. just to see whether to get the result? 
I know we have gas price in block header from rpc, but we do not have that in nearlib block header. So really bother me how to write test

Reply:
I think it should be enough to check that the value is there, is numeric and is greater than 0.

Reply:
Since @frol's change that exposes config is already merged, we can check that this value is at least minimum gas price.

Reply:
@bowenwang1996 note that we also need this change in both `ci` and `ci-staging` nets, not only merged

Reply:
what is at least minimum gas price? @bowenwang1996

Reply:
@bowenwang1996 That one is not even implemented in nearlib yet, and only available on the master branch, so it seems like a chicken and an egg problem here.

Reply:
Okay we can add that later.